dist/getFilepathsFromGlob.js

```diff
@@ -22,6 +22,7 @@ function getFilepathsFromGlob(baseDir, options) {
     cwd: baseDir,
     bashNative: [],
     onlyFiles: true,
+    dot: true, // match behavior of watchman from relay-compiler
```

Comment:
I've been putting many shared fragments in a file with no name, e.g. `mycomponent/.fragments.js`.
The standard relay compiler picks this up since when using watchman config, so this brings its behaviour in line.